NEW-STANDARD.md

```diff
@@ -0,0 +1,12 @@
+When adding a new WHATWG standard, these are the things that need to be done:
+
+1. Identify a shortname that is used for the spec.whatwg.org subdomain and repository (for tooling it is important that these are named identically).
+   1. Ensure the subdomain works and is configured: https://github.com/whatwg/misc-server.
+   1. Ensure there is repository.
+1. Create an SVG logo in the right shade of green (i.e., WHATWG green) and add it to https://github.com/whatwg/whatwg.org/tree/main/resources.whatwg.org.
+1. Create a Twitter account.
+   1. Use https://github.com/whatwg/whattweetbot-keys to get the relevant keys. (The Steering Group will help with this.)
+1. Add it to https://github.com/whatwg/sg/blob/main/db.json and ensure the relevant details are filled in. (In case there's a new Workstream more updates will be needed.)
+1. Run https://github.com/whatwg/spec-factory.
```

Comment:
```suggestion
1. Run https://github.com/whatwg/spec-factory inside a local checkout of the standard's repository to create the appropriate boilerplate files.
```

although maybe this should be done before creating the repository...

Reply:
I don't think we support this option currently, but that might be good to add.

Reply:
Oh, I see. I guess leaving as-is then is fine and people will have to read the docs over there to realize that they should run it in the parent directory (?).

Reply:
I guess I should look at this again (prolly tomorrow) and divide steps a bit between Admin/SG and editor. It's mainly meant as a checklist for Admin/SG, but that is not clear.